defaults/main.yml

```
@@ -5,6 +5,8 @@ configure_registry: false
 remove_persistent_data: false
 remove_existing_container: false
 persistent_data_path: /opt/portainer:/data
+container_image: portainer/portainer:{{ version }}
+container_name: portainer 
```

One suggestion: remove that trailing whitespace.

```suggestion
container_name: portainer
```